src/components/new-asset-form/new-asset.tsx

```diff
@@ -233,26 +232,14 @@ export const NewAsset = ({
                   {renderAssetTypeOptions()}
                 </Field>
               </div>
-              {values.assetType !== "Estate" && (
-                <InlineAssetSearch
-                  assetTypes={["Estate"]}
-                  name="propertyEstate"
-                  label="Estate this property is in"
-                  onChange={handleChange}
-                  setFieldValue={setFieldValue}
-                  value={values.propertyEstate || ""}
-                />
-              )}
-              {values.assetType !== "Block" && values.assetType !== "Estate" && (
-                <InlineAssetSearch
-                  assetTypes={["Block"]}
-                  name="propertyBlock"
-                  label="Block this property is in"
-                  onChange={handleChange}
-                  setFieldValue={setFieldValue}
-                  value={values.propertyBlock || ""}
-                />
-              )}
+              <InlineAssetSearch
+                assetTypes={[]}
```

Comment:
Previously, we allowed either block or estate. Would it make sense to specify these two in the assetTypes array?

Reply:
I've asked this in the chat (team blue chat) and while I was of the opinion that only certain asset types could be parent assets (like block, estate, walkup block etc), I was told that we should give the user freedom in regards to the choice of the parent asset.